fastlane_core/lib/fastlane_core/tool_collector.rb

```diff
@@ -57,6 +92,10 @@ def launches
       @launches ||= Hash.new(0)
     end
 
+    def versions
```

Comment:
It would be great to have an example hash as a comment above to get a better feel of how the data is stored, and what kind of data it is